src/screens/events/SetGeneralEventDetails.tsx

```diff
@@ -41,6 +41,7 @@ const SetGeneralEventDetails = ({ navigation }: EventProps) => {
     const [endTime, setEndTime] = useState<Timestamp | undefined>(event.endTime ?? undefined);
     const [description, setDescription] = useState<string>("");
     const [coverImageURI, setCoverImageURI] = useState<string>();
+    const [general, setIsGeneral] = useState<boolean>(event.general ?? false);
```

Comment:
change general variable to isGeneral 
(see lines above, second variable has "set" in front of first variable name)